nms/server/host/routes.ts

```diff
@@ -117,10 +117,12 @@ router.post(
       res,
     ) => {
       const featureId = req.params.featureId;
-      const results: Record<FeatureID, FeatureConfig> & {
-        config?: FeatureFlagConfig;
-      } = {...featureConfigs};
-      results.config = {};
+      const results: Record<
```

Comment:
Maybe the code becomes a tiny bit less awkward if start out with one feature flag right away.
I.E.
```ts
const result: FeatureConfig & {
   config?: FeatureFlagConfig;
} = featureConfigs[featureId];
```